reactor-core/src/test/java/reactor/core/publisher/FluxIterableTest.java

```diff
@@ -44,11 +50,104 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.mockito.Mockito.mock;
 
 public class FluxIterableTest {
 
 	final Iterable<Integer> source = Arrays.asList(1, 2, 3, 4, 5, 6, 7, 8, 9, 10);
 
+	static Stream<Function<Flux, Flux>> factory() {
```

Comment:
Consider not duplicating the code present in both this file and the flatten scenario. Otherwise, best to add a note as a comment that these should be kept in sync upon change.